tests/utils/test_gower_distance.py

```diff
@@ -88,3 +57,26 @@ def test_mix_columns_full_matrix_with_nulls():
     assert_that(dist[-1], has_length(data.shape[0]))
     assert_that(dist[3], has_item(greater_than(0.01)))
     assert_that(min(dist[0]), less_than_or_equal_to(0))
+
+
+def test_mix_columns_nn_matrix_with_nulls_vectorized():
```

Comment:
Good idea, are gowers dependencies OK?

Reply:
The dependencies were only added to dev-requirements so as far i understood are not downloaded with the package, am I correct? @matanper

Reply:
Yep it's right, just making sure that the testing env will be functional.